tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -1884,4 +1907,118 @@ public Multimap<String, Continuation> getContinuations() {
     public Map<String, String> getBaseUnitToStatus() {
         return baseUnitToStatus;
     }
+
+    static final Rational LIMIT_UPPER_RELATED = Rational.of(10000);
+    static final Rational LIMIT_LOWER_RELATED = LIMIT_UPPER_RELATED.reciprocal();
+
+    public Map<Rational, String> getRelatedExamples(
+            String inputUnit, Set<UnitSystem> allowedSystems) {
+        Set<String> others = new LinkedHashSet<>(canConvertBetween(inputUnit));
+        if (others.size() <= 1) {
+            return Map.of();
+        }
+        // add common units
+        if (others.contains("meter")) {
+            others.add("kilometer");
+            others.add("millimeter");
+        } else if (others.contains("liter")) {
+            others.add("milliliter");
+        }
+        // remove unusual units
+        others.removeAll(
+                Set.of(
+                        "fathom",
+                        "carat",
+                        "grain",
+                        "slug",
+                        "drop",
+                        "pinch",
+                        "cup-metric",
+                        "dram",
+                        "jigger",
+                        "pint-metric",
+                        "bushel, barrel",
+                        "dunam",
+                        "rod",
+                        "chain",
+                        "furlong",
+                        "fortnight",
+                        "rankine",
+                        "kelvin",
+                        "calorie-it",
+                        "british-thermal-unit-it",
+                        "foodcalorie"));
```

Comment:
should be driven by preferences? or other metadata?

Reply:
We don't have any of that information available.

Remember, these are just examples to help establish the meaning of the current unit, so the goal was to quickly eliminate units that would not be very familiar to lots of people. I did, however, notice a problem above with bushel, bushel, barrel so will fix.